build.gradle

```diff
@@ -35,6 +35,12 @@ tasks.withType(Test) {
             systemProperty prop.key, prop.value
         }
     }
+    doFirst {
```

Comment:
Yes, I made a note in the commit comments, but I should add a line in the gradle file itself.